src/main/java/me/figo/FigoConnection.java

```diff
@@ -211,18 +111,13 @@ protected Gson createGson() {
      */
     public String getLoginUrl(String scope, String state) {
         try {
-            StringBuilder sb = new StringBuilder();
-            sb.append(apiEndpoint);
-            sb.append("/auth/code?response_type=code&client_id=");
-            sb.append(URLEncoder.encode(this.clientId, "ISO-8859-1"));
-            sb.append("&redirect_uri=");
-            sb.append(URLEncoder.encode(this.redirectUri, "ISO-8859-1"));
-            sb.append("&scope=");
-            sb.append(URLEncoder.encode(scope, "ISO-8859-1"));
-            sb.append("&state=");
-            sb.append(URLEncoder.encode(state, "ISO-8859-1"));
-            return sb.toString();
+            return getApiEndpoint()
```

Comment:
I prefer to not instantiate 10 objects (StringBuilders and the number is only swag) when one would be enough